packages/sado-connect/src/providers/SadoContext.tsx

```diff
@@ -11,6 +11,28 @@ interface SadoContextI {
 
 const SadoContext = createContext<SadoContextI>(undefined as any);
 
+/**
+ * (Optionally) global context provider for SadoConnectKit and its consumer(s).
```

Comment:
do you mean `Optional`?

Reply:
(Optionally) global / Global (optional)

The grammar is correct.